core-java-modules/core-java-networking-3/src/main/java/com/baeldung/timeout/TcpServerSocket.java

```diff
@@ -3,6 +3,10 @@
 import java.io.BufferedReader;
 import java.io.IOException;
 import java.io.InputStreamReader;
+<<<<<<< HEAD
+//A Java program for a Server
+=======
+>>>>>>> upstream/master
```

Comment:
Note: there's a strange bit of git merge conflict here that needs resolving.

Reply:
i don't know what happened here. This is from my previous article. I removed the comment